util/chplenv/chpl_dlmalloc_configs.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import utils
+from utils import memoize
```

Comment:
Were you planning to memoize the functions in these files? It seems like it would be safe to do so. E.g.

```python
@memoize
def get_uniq_cfg_path():
    ...
```